Gems/LmbrCentral/Code/Source/Audio/AudioTriggerComponent.cpp

```diff
@@ -68,8 +77,13 @@ namespace LmbrCentral
                 ->Event("SetObstructionType", &AudioTriggerComponentRequestBus::Events::SetObstructionType)
                 ;
 
-            behaviorContext->EBus<AudioTriggerComponentNotificationBus>("AudioTriggerComponentNotificationBus")
-                ->Handler<BehaviorAudioTriggerComponentNotificationBusHandler>()
+            behaviorContext->Class<Audio::TriggerNotificationIdType>("TriggerNotificationIdType")
```

Comment:
Seems like this name could conflict with a PhysX Volume Trigger Id Type.
I doubt in the future there would be a specific type called "TriggerNotificationIdType" for another type of system, so this would be fine, but prefixing the BehaviorContext name with Audio might shave a few seconds off understanding the type in the ScriptCanvas Editor if one sees "AudioTriggerNotificationIdType".

Reply:
Will update.